src/env.mjs

```diff
@@ -71,6 +71,8 @@ export const env = createEnv({
             });
           }),
       ),
+    NEXT_PUBLIC_AC_TELEGRAM_LINK: z.string(),
+    NEXT_PUBLIC_AC_HELPDESK_LINK: z.string(),
```

Comment:
question: since we know ahead of time that these are telegram links, do you think we can validate that they are indeed of that structure with zod's [`z.string().url()`](https://zod.dev/?id=strings) validator?

Reply:
Hi David, thanks for the comment and yes I think that makes sense. Made the change @ [2ae9a77](https://github.com/AfterClass-io/afterclass.io-v2/pull/231/commits/2ae9a77f2b75843b0ed3bf02f4e8e4718903551a) !